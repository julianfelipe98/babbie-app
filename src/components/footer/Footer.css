#logo-img{
    width: "30%";
    height: "auto";
}
.footer{
    display: flex;
    justify-content: space-evenly;
    background-color: #ffff;
    padding: 0.5% 1% 
}
.center-align{
    align-self: center;
}
#icons a{
    padding-right: 15px;
    color: #5271ff;
}
#copyright{
    font-family: Georgia, 'Times New Roman', Times, serif;
    color: #5C5C5C;
}