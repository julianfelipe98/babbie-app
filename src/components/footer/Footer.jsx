import React from "react";
import logo from "../../assets/logo.svg";
import FacebookIcon from "@material-ui/icons/Facebook";
import TwitterIcon from "@material-ui/icons/Twitter";
import InstagramIcon from "@material-ui/icons/Instagram";
import "./Footer.css";
export const Footer = () => {
  return (
    <div className="footer">
      <img id="logo-img" src={logo} alt="logo-babbie"></img>
      <div id="icons" className="center-align">
        <a href="https://www.facebook.com">
          <FacebookIcon fontSize="large" />
        </a>
        <a href="https://www.instagram.com">
          <InstagramIcon fontSize="large" />
        </a>
        <a href="https://www.twitter.com">
          <TwitterIcon fontSize="large" />
        </a>
      </div>
      <div id="copyright" className="center-align">
        <p>Copyright 2021 BabbieAppÂ®</p>
      </div>
    </div>
  );
};
