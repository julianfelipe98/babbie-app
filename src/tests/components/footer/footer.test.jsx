import { shallow } from "enzyme";
import { Footer } from "../../../components/footer/Footer";

describe("Footer component tests", () => {
  test("should render <Footer/> in the correct way", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
