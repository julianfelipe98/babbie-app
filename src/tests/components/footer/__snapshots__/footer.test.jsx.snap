// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component tests should render <Footer/> in the correct way 1`] = `
<div
  className="footer"
>
  <img
    alt="logo-babbie"
    id="logo-img"
    src="logo.svg"
  />
  <div
    className="center-align"
    id="icons"
  >
    <a
      href="https://www.facebook.com"
    >
      <Memo([object Object])
        fontSize="large"
      />
    </a>
    <a
      href="https://www.instagram.com"
    >
      <Memo([object Object])
        fontSize="large"
      />
    </a>
    <a
      href="https://www.twitter.com"
    >
      <Memo([object Object])
        fontSize="large"
      />
    </a>
  </div>
  <div
    className="center-align"
    id="copyright"
  >
    <p>
      Copyright 2021 BabbieAppÂ®
    </p>
  </div>
</div>
`;
